packages/generator/src/service-mapping.ts

```diff
@@ -1,5 +1,6 @@
 import { parse } from 'path';
-import { unixEOL, createLogger, readJSON } from '@sap-cloud-sdk/util';
```

Comment:
[pp] Rename the full file `service-mapping`should go away completely as a term.

Reply:
Removed all occurrences of the term.